src/libraries/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoTests.cs

```diff
@@ -176,6 +176,11 @@ public void TestFirstYearOfJapaneseEra()
             DateTime dt = new DateTime(1989, 01, 08); // Start of Heisei Era
 
             string formattedDateWithGannen = "\u5E73\u6210 \u5143\u5E74 01\u6708 08\u65E5";
+            if (PlatformDetection.IsBrowser)
+            {
+                formattedDateWithGannen = "Heisei \u5143\u5E74 01\u6708 08\u65E5";
```

Comment:
in the newer ICU data we had to give up on native names for calendars but probably we need undo it for japanese one.